app/crawler/jtbc/api.py

```diff
@@ -0,0 +1,59 @@
+from datetime import datetime
+from typing import List
+
+import aiohttp
+
+from app.crawler.base import Article, BaseNewsCrawler
+from common.utils.logger import get_logger
+
+logger = get_logger(__name__)
+
+
+class JTBCNewsApiCrawler(BaseNewsCrawler):
+    """
+    JTBC 뉴스 API 크롤러
+    """
+
+    BASE_URL = "https://news-api.jtbc.co.kr/v1/get/contents/section/list/articles"
+
+    async def fetch_articles(self) -> List[Article]:
+        logger.info("[JTBC] 뉴스 API 요청 시작")
+
+        articles: List[Article] = []
+
+        # 오늘 날짜 가져오기
+        today = datetime.now().strftime("%Y-%m-%d")
+        logger.info(f"[JTBC] 오늘 날짜: {today}")
+
+        timeout = aiohttp.ClientTimeout(total=30)
+        connector = aiohttp.TCPConnector(limit=10, ssl=False)
+
+        try:
+            async with aiohttp.ClientSession(
+                connector=connector, timeout=timeout
+            ) as session:
+                params = {"pageNo": 1, "pageSize": 20, "articleListType": "ARTICLE"}
+
+                async with session.get(self.BASE_URL, params=params) as response:
+                    if response.status == 200:
```

Comment:
분기가 많아지면서 depth가 너무 커지고있습니다.


```suggestion
if response.status != 200:
    logger.error(f"요청 실패 - 상태 코드: {response.status}")
    raise # or return ...

...
```

early return 을 적용해볼까요?

Reply:
early return 패턴을 항상 생각하면서 코딩하겠습니다. 
일단 구현이 목표가 되니, 가독성 부분을 자꾸 놓치게 되는거 같습니다.